templates/weblogic.service.j2

```diff
@@ -0,0 +1,18 @@
+[Unit]
```

Comment:
Will this file get overwritten during bootstrap to configure specific WL flavours?

Reply:
No, only the variables in this file are changed. Ideally i'd like to use the domain_name variable so the file is dynamic for different weblogic domains, instead of hardcoded to NDelius. THat appears to be set in the packer install.

Reply:
Ok, I'd overwrite this in bootstrap to be sure and then start and enable the service. Did similar in Alfresco myself as some bit are only known when the instance is deployed. Something to think about